docs/source/models/gbd2017/causes/neonatal_jaundice/index.rst

```diff
@@ -7,24 +7,44 @@ Haemolytic disease and other neonatal jaundice
 Disease Description
 -------------------
 
-Case definition in GBD 2017
-+++++++++++++++++++++++++++
+`Hemolytic disease`_ and `neonatal jaundice`_ are two separate etiologies by which
+an newborn infant might develop hyperbilirubinemia, which can in turn become
+kernicterus.
 
+Hemolytic disease is a product of the mother's immunoglobin G antibodies
+attacking the fetus or newborn's red blood cells. The main presentations of
+this condition are RhD hemolytic diseaes and ABO hemolytic disease; however
+there are over 30 other blood groups, various combinations of which can also
+result in hemolytic disease.
 
+Neonatal jaundice is a product of total serum or plasma bilirubin (TB)
+levels rising too high.
 
-`Haemolytic disease of the newborn`_ and other `neonatal jaundice`_ refers to
-several aetiologies by which an infant develops extreme hyperbilirubinemia_
-(EHB) and can then go on to develop kernicterus_. The aetiologies that we model
-for GBD are EHB from `Rhesus (Rh) disease`_, preterm birth,
-`glucose-6-phosphate dehydrogenase deficiency`_ (G6PD), and other causes.
+Bilirubin is produced during the breakdown of hemoglobin and other
+heme-containing proteins, and healthy infants commonly have TB levels of > 1
+mg/dL, which increases in the hours after birth.
 
-.. _Haemolytic disease of the newborn: https://www.urmc.rochester.edu/encyclopedia/content.aspx?ContentTypeID=90&ContentID=P02368
-.. _neonatal jaundice: https://en.wikipedia.org/wiki/Neonatal_jaundice
-.. _hyperbilirubinemia: https://www.chop.edu/conditions-diseases/hyperbilirubinemia-and-jaundice
-.. _kernicterus: https://en.wikipedia.org/wiki/Kernicterus
+At lower TB levels, an infant may be diagnosed with benign hyperbilirubinemia,
+typified by a yellow pigmentation in the newborn caused by the bilirubin.
+However as TB level increases towards extreme hyperbilirubinemia (EHB; typically
+classified as TB > 30 mg/dL), the infant becomes increasingly at risk of
+bilirubin crossing  the blood-brain barrier and binding to brain tissue, 
+resulting in brain damage: bilirubin-induced neurologic dysfunction (BIND).
+
+The most common etiologies of hyperbilirubinemia include incompatibilty between
+the mother and infant's blood types, lack of breastfeeding, recieving breastmilk 
+if the mother's antigens are incompatible with the infant (called Breastmilk
+jaundice; this would develop a few days after birth), and poor liver function in
+the infant.
+
+**The etiologies modeled by GBD** include EHB from `Rhesus (Rh) disease`_, preterm
+birth, `glucose-6-phosphate dehydrogenase deficiency`_ (G6PD), and other causes.
```

Comment:
It looks like G6PD is a genetic condition which is not captured in the etiologies you describe above, and Rh disease is a subset of hemolytic disease, if I'm understanding correctly. We should probably also explain how preterm birth leads to jaundice, and we should probably ask the GBD modelers what they mean by "other causes."

Reply:
I've updated to describe all etiologies modeled by GBD, and i've put in a todo my guess (from reading the appendix) what "other causes" is. I've messaged helena to confirm/deny that guess. @NathanielBlairStahn